atoma-proxy-service/src/components/openapi.rs

```diff
@@ -95,7 +95,7 @@ pub fn openapi_router() -> Router {
                 components.add_security_scheme(
                     "bearerAuth",
                     SecurityScheme::Http(Http::new(HttpAuthScheme::Bearer)),
-                )
+                );
```

Comment:
I thought the clippy would be happy with the former changes, as it is redundant and verbose to add the `;`

Reply:
The semicolon might be optional but when extending the block with new code, it doesn’t require a change in previous last line.